src/main/java/seedu/duke/component/Resistor.java

```diff
@@ -0,0 +1,18 @@
+package seedu.duke.component;
+
+public class Resistor extends Component {
+    private static final String SI_UNIT = "μH";
+
+    public Resistor(double value) {
+        super(value);
+    }
+    /**
```

Comment:
If you look in the build error, it says `'METHOD_DEF' should be separated from previous statement. [EmptyLineSeparator]`. So just have to add an empty line.
```suggestion

    /**
```